src/ui/EditorNS/editor.cpp

```diff
@@ -656,7 +708,7 @@ namespace EditorNS
     void Editor::print(QPrinter *printer)
     {
         sendMessage("C_CMD_DISPLAY_PRINT_STYLE");
-        m_webView->print(printer);
+        //m_webView->print(printer); // FIXME
```

Comment:
There is now print support for WebEngineView:

[void QWebEnginePage::print(QPrinter *printer, FunctorOrLambda resultCallback);](http://doc-snapshots.qt.io/qt5-5.8/qwebenginepage.html#print)

However, it's only starting at Qt5.8 so we'd have a rather high Qt requirement if we're to use it.